doc/user/layouts/_default/baseof.html

```diff
@@ -77,10 +77,24 @@
     <div class="backdrop"></div>
 
     <script>
+      /* Add anchor links to headings */
       anchors.add(
         ".content h2, .content h3, .content h4, .content h5, .content h6"
       );
 
+      /* Copy anchor links to clipboard on click */
+      $('.anchorjs-link').click(function (e) {
+        e.preventDefault();
+        var copyText = $(this).prop('href');
+
+        document.addEventListener('copy', function(e) {
+          e.clipboardData.setData('text/plain', copyText);
+          e.preventDefault();
```

Comment:
Thanks @benesch for bringing this up. 

@jseldess Since this event listener will be re-created/re-added every time a link is clicked so it makes sense to clean up the memory once the event has done its job.